### ISSUE TITLE
Error when listening to Usb connect events in electron

### ISSUE THREAD

**Issue by akashdeep-sharma** [node-usb/node-usb#597]
## Bug Description:
Getting this error 
`Uncaught Exception:
Error: No native build was found for platform=darwin arch=arm64 runtime=electron abi=114 uv=1 armv=8 libc=glibc node=18.14.0 electron=24.1.2 webpack=true
 `
 

## Steps to Reproduce:
1. using WebUsb with allowedAllDevices set to true
2.Listening to 'connect ' and 'disconnect' events 
3.getting this error after building electron app and running it. App works fine when I don't listen to events from usb package 

`  const webUsb = new usb.WebUSB({
      allowAllDevices: true,
    });
    webUsb.addEventListener('connect', (event) => {
      console.log(event);
    });`
code snippet

**Comment by thegecko** [comment]
There should be a valid prebuild for your environment, are you trying to rebuild the package in any way?

**Comment by akashdeep-sharma** [comment]
Yes I tried rebuilding
this is my package command
`    "package": "ts-node ./.erb/scripts/clean.js dist && npm run build && electron-builder build --publish never && electron-rebuild -w usb",
`

**Comment by thegecko** [comment]
Yes, don't rebuild the package. The correct prebuild should already be shipped. Rebuilding it causes issues

**Comment by akashdeep-sharma** [comment]
Without rebuilding also I am getting same error

**Comment by thegecko** [comment]
OK, can you confirm you see the same issue with our electron test-harness:

https://github.com/node-usb/node-usb-example-electron
